framework/src/meshgenerators/MeshRepairGenerator.C

```diff
@@ -37,6 +37,10 @@ MeshRepairGenerator::validParams()
                         false,
                         "Create new blocks if multiple element types are present in a block");
 
+  params.addParam<bool>("merge_boundary_ids_with_same_name",
+                        false,
```

Comment:
Default to false ... but when do we *not* want to do this?  We treat names as first-class identifiers in MOOSE, and we *only* treat numeric ids as first-class identifiers in libMesh, so if we ever have a single name matching multiple ids isn't that just a bug waiting to happen?

Reply:
The reason behind this is that when I looked at the object it looked like every other manipulation defaulted to false. It feels like to control what this guy does you have to manually set the operation to true. So I just went with the current design.

Reply:
Ah, that's right; this is the `MeshRepairGenerator`, we're trying to be as non-intrusive as possible here.  Question rescinded.